sampler.go

```diff
@@ -26,6 +26,9 @@ type Sampler interface {
 
 	// Gauge sets the gauge with the given name to the given value
 	Gauge(name string, value int, tags ...Tag) error
+
+	// Time sets the time in milliseconds with the given name to the given value
+	Time(name string, value int, tags ...Tag) error
```

Comment:
I suggest using the time.Duration type. It will make the intention more explicit and also decouple the caller from details if it is on ms/sec/etc (normalize in ns and them convert internally to what you want).

Also time.Duration has a better String method to represent it when debugging

Reply:
indeed..  much better use time.Duration..
i didn't know Printf behavior with time.Duration.. very cool.
thanks :)